milli/src/update/settings.rs

```diff
@@ -91,6 +93,18 @@ impl<'a, 't, 'u, 'i> Settings<'a, 't, 'u, 'i> {
         self.criteria = Some(Some(criteria));
     }
 
+    pub fn reset_stop_words(&mut self) {
+        self.stop_words = Some(None);
+    }
+
+    pub fn set_stop_words(&mut self, stop_words: BTreeSet<String>) {
+        if stop_words.is_empty() {
+            self.stop_words = Some(None);
+        } else {
+            self.stop_words = Some(Some(stop_words));
+        }
```

Comment:
```suggestion
        self.stop_words = if stop_words.is_empty() {
            Some(None)
        } else {
            Some(Some(stop_words));
        };
```

I am picky, but could you...? 😏

Reply:
:angry: